tests/integration/api/test_geti_interaction.py

```diff
@@ -0,0 +1,223 @@
+
+    def test_export_and_infer_openvino(self):
+        """Test exporting the model to OpenVINO."""
+        for precision in [OTXPrecisionType.FP16, OTXPrecisionType.FP32]:
+            exported_path = self.engine.export(
+                export_format=OTXExportFormatType.OPENVINO,
+                export_precision=precision,
+                explain=(precision == OTXPrecisionType.FP32),
+                export_demo_package=True,
+            )
+            export_dir = exported_path.parent
+            assert export_dir.exists()
+
+            # Test Model API model
+            if self.mapi_model_class is not None:
+                ov_export_dir = self.tmp_path / "ov_export"
+                ov_export_dir.mkdir(parents=True, exist_ok=True)
+                unzip_exportable_code(
+                    work_dir=self.tmp_path,
+                    exported_path=exported_path,
+                    dst_dir=ov_export_dir,
+                )
+                xml_path = ov_export_dir / "exported_model.xml"
+                mapi_model = self.mapi_model_class.create_model(xml_path)
+                assert mapi_model is not None
+
+                predictions = mapi_model(self.image)
+                assert predictions is not None
+                assert len(predictions.top_labels) > 0
+
+            exported_path.unlink(missing_ok=True)
+
+    def test_optimize_openvino_fp32(self):
+        """Test optimizing the OpenVINO model with FP32 precision."""
+        fp32_export_dir = self.tmp_path / "fp32_export"
+        fp32_export_dir.mkdir(parents=True, exist_ok=True)
+        exported_path = self.engine.export(
+            export_format=OTXExportFormatType.OPENVINO,
+            export_precision=OTXPrecisionType.FP32,
+            explain=True,
+            export_demo_package=True,
+        )
+        unzip_exportable_code(
+            work_dir=self.tmp_path,
+            exported_path=exported_path,
+            dst_dir=fp32_export_dir,
+        )
+        optimized_path = self.engine.optimize(
+            checkpoint=fp32_export_dir / "exported_model.xml",
+            export_demo_package=True,
+        )
+        assert optimized_path.exists()
+
+        # Test Model API model
+        if self.mapi_model_class is not None:
+            ov_optimized_dir = self.tmp_path / "ov_optimize"
+            ov_optimized_dir.mkdir(parents=True, exist_ok=True)
+            unzip_exportable_code(
+                work_dir=self.tmp_path,
+                exported_path=optimized_path,
+                dst_dir=ov_optimized_dir,
+            )
+            xml_path = ov_optimized_dir / "exported_model.xml"
+            mapi_model = self.mapi_model_class.create_model(xml_path)
+            assert mapi_model is not None
+
+            predictions = mapi_model(self.image)
+            assert predictions is not None
+            assert len(predictions.top_labels) > 0
+
+
+@pytest.mark.parametrize("task", pytest.TASK_LIST)
```

Comment:
I think we don't need all models from list for API test, just 3 models (fast/balance/accurate) should suffic?

Reply:
Fail point, currently I implemented only one model per task, it is effnet_b0 (I guess it is "fast").
But this can be extended to fast/balance/accurate.

Reply:
yeah, not sure how we'll address this. But currently config.json is tied to one speific model. To be able to train all 3, we need to decide if we wanna have 3 config.json or overwrite model entry in current config.json.